src/Tools/dotnet-counters/Exporters/ConsoleWriter.cs

```diff
@@ -140,22 +149,39 @@ public void AssignRowsAndInitializeDisplay()
             foreach (ObservedProvider provider in _providers.Values.OrderBy(p => p.KnownProvider == null).ThenBy(p => p.Name)) // Known providers first.
             {
                 Console.WriteLine($"[{provider.Name}]"); row++;
+                int rowCount = 0;
+
                 foreach (ObservedCounter counter in provider.Counters.Values.OrderBy(c => c.DisplayName))
                 {
+                    if(rowCount >= _consoleHeight - 5)
```

Comment:
Is this code trying to prevent us from displaying more counters than we have vertical space available? If so should it be refering to row rather than rowCount? For example I could have 100 providers and each provider has 1 counter in it which means rowCount would always be zero.

Reply:
You're right. Made the change to reference row:

```c#
if(row >= _consoleHeight){
    break;
}
```